catroid/src/main/res/values/strings.xml

```diff
@@ -1156,7 +1153,7 @@
 
 
     <!-- Control bricks -->
-    <string name="brick_when_started">When program starts</string>
+    <string name="brick_when_started">When project or scene starts</string>
```

Comment:
```suggestion
    <string name="brick_when_started">When scene starts</string>
```
As I wrote in my longer comment, I strongly believe that "When scene starts" is a better name than "When project starts" (and "When project or scene starts", which is clumsy and not precise) for that brick: Not only does it perfectly fit the metaphor, even in theater pieces or movies with a _single_ scene, it also emphasizes the new concept of scenes we have introduced. It thereby pushes the concept of our scenes to the forefront of awareness among our users, which is a good thing as scenes are a bit difficult to discover as of now. Nevertheless, we still want to keep scenes a bit more in the background for simpler projects with single scenes, just hinting at "something more" through this new name of the "When scene starts" of the old "When project starts" brick. Scenes can thus be discovered by more advanced users, but with a clear meaning nevertheless even for users that do not get any additional explanation about scenes. Intuitiveness at its best ;-). It also captures much better the intended meaning for any _later_ scenes than the currently "When project starts" brick, which is currently named incorrectly in these second, third, etc scenes.